fanficfare/adapters/adapter_royalroadcom.py

```diff
@@ -129,7 +129,7 @@ def make_soup(self, data):
         for style_element in style_elements:
             style_content = style_element.string
             if style_content and 'display: none;' in style_content.lower():
-                class_matches = re.findall(r'\.(\S+)\s*\{\s*display:\s*none;\s*\}', style_content)
+                class_matches = re.findall(r'\.(\S+)\s*\{\s*display:\s*none;\s*(?:speak:\s*never;\s*)?\}', style_content)
```

Comment:
Why not
```suggestion
                class_matches = re.findall(r'\.(\S+)\s*\{[^\}]*display\s*:\s*none\s*;[^\}]*\}', style_content, flags=re.IGNORECASE)
```
?
And maybe do a more robust check using `display\s*:\s*none\s*;` for `style_content.lower()`?

Reply:
Be my guest. The code is garbage . I'm an SI guy not a programmer. The only reason I'm even programing is because this affected me personally. 
IF you're doing ignorecase you should change "style_content.lower()" in 131 to style_content. just pushed with that